client/src/App.jsx

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
-import theme from './theme';
 import { ThemeProvider, Button } from '@material-ui/core';
-import store from './store';
 import { Provider } from 'react-redux';
+import theme from './theme';
+import store from './store';
 import Main from './components/Main/Main';
 import Header from './components/Header/Header';
 
 const App = () => {
-  console.log('STORE', store);
```

Comment:
not major but don't need console.logs ;D

Reply:
is that in an old commit? i dont have that on my branch

Reply:
that's a removed line. will resolve conversation